[release-blocker] Kubernetes 1.15 stages are failing due to go1.13 variable usage

**What happened**:
Some go1.13 variable changes in `pkg/util/common.go` caused the 1.15 mock stage to fail (since it's using kube-cross on go1.12).

```shell
$ ./gcbmgr stage release-1.15 --official --buildversion=$(curl -Ls https://dl.k8s.io/ci/latest-1.15.txt)
```

```
gcbmgr: BEGIN main on auggievmw01 Tue 11 Feb 2020 10:18:04 AM EST

Checking required system packages: OK
Checking /home/augustus/go/src/k8s.io/release state: OK
Checking/setting cloud tools: OK

Trying to get kubecross version for release-1.15 ... OK: v1.12.12-1
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
STAGE b3f768e9-bf24-493e-b23e-520154da409c submitted successfully.
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

To view last build:
$ gcbmgr tail

To view this specific build:
$ gcbmgr tail b3f768e9-bf24-493e-b23e-520154da409c
-OR-
$ /usr/bin/gcloud --project kubernetes-release-test builds log --stream b3f768e9-bf24-493e-b23e-520154da409c
-OR-
https://console.cloud.google.com/gcr/builds/b3f768e9-bf24-493e-b23e-520154da409c?project=648026197307

gcbmgr: DONE main on auggievmw01 Tue 11 Feb 2020 10:18:07 AM EST in 3s
```

Here's the GCB run output:

<details>

```shell
starting build "b3f768e9-bf24-493e-b23e-520154da409c"

FETCHSOURCE
BUILD
Starting Step #0
Step #0: Already have image (with digest): gcr.io/cloud-builders/git
Step #0: Cloning into 'release'...
Finished Step #0
Starting Step #1
Step #1: Pulling image: k8s.gcr.io/kube-cross:v1.12.12-1
Step #1: v1.12.12-1: Pulling from kube-cross
Step #1: c7b7d16361e0: Pulling fs layer
Step #1: b7a128769df1: Pulling fs layer
Step #1: 1128949d0793: Pulling fs layer
Step #1: 667692510b70: Pulling fs layer
Step #1: c70d80036479: Pulling fs layer
Step #1: 1de3250885d9: Pulling fs layer
Step #1: 5edd1231cc13: Pulling fs layer
Step #1: de8369c9dc4d: Pulling fs layer
Step #1: 4362946f4289: Pulling fs layer
Step #1: 667692510b70: Waiting
Step #1: a84c2153261f: Pulling fs layer
Step #1: 04195b6db89d: Pulling fs layer
Step #1: c70d80036479: Waiting
Step #1: ebd61dd83553: Pulling fs layer
Step #1: 591054f206a5: Pulling fs layer
Step #1: bceeb98e2162: Pulling fs layer
Step #1: 5edd1231cc13: Waiting
Step #1: de8369c9dc4d: Waiting
Step #1: a84c2153261f: Waiting
Step #1: 1de3250885d9: Waiting
Step #1: 591054f206a5: Waiting
Step #1: 04195b6db89d: Waiting
Step #1: ebd61dd83553: Waiting
Step #1: bceeb98e2162: Waiting
Step #1: b7a128769df1: Verifying Checksum
Step #1: b7a128769df1: Download complete
Step #1: 1128949d0793: Verifying Checksum
Step #1: 1128949d0793: Download complete
Step #1: c7b7d16361e0: Verifying Checksum
Step #1: c7b7d16361e0: Download complete
Step #1: 667692510b70: Verifying Checksum
Step #1: 667692510b70: Download complete
Step #1: 5edd1231cc13: Verifying Checksum
Step #1: 5edd1231cc13: Download complete
Step #1: c70d80036479: Verifying Checksum
Step #1: c70d80036479: Download complete
Step #1: 4362946f4289: Verifying Checksum
Step #1: 4362946f4289: Download complete
Step #1: c7b7d16361e0: Pull complete
Step #1: b7a128769df1: Pull complete
Step #1: 1de3250885d9: Verifying Checksum
Step #1: 1de3250885d9: Download complete
Step #1: 1128949d0793: Pull complete
Step #1: 04195b6db89d: Download complete
Step #1: ebd61dd83553: Download complete
Step #1: de8369c9dc4d: Verifying Checksum
Step #1: de8369c9dc4d: Download complete
Step #1: 591054f206a5: Verifying Checksum
Step #1: 591054f206a5: Download complete
Step #1: bceeb98e2162: Verifying Checksum
Step #1: bceeb98e2162: Download complete
Step #1: 667692510b70: Pull complete
Step #1: a84c2153261f: Verifying Checksum
Step #1: a84c2153261f: Download complete
Step #1: c70d80036479: Pull complete
Step #1: 1de3250885d9: Pull complete
Step #1: 5edd1231cc13: Pull complete
Step #1: de8369c9dc4d: Pull complete
Step #1: 4362946f4289: Pull complete
Step #1: a84c2153261f: Pull complete
Step #1: 04195b6db89d: Pull complete
Step #1: ebd61dd83553: Pull complete
Step #1: 591054f206a5: Pull complete
Step #1: bceeb98e2162: Pull complete
Step #1: Digest: sha256:97e77164ddc1f9ab56bcbab3e2895b882097b1eabd820497e894f41c0e4908fe
Step #1: Status: Downloaded newer image for k8s.gcr.io/kube-cross:v1.12.12-1
Step #1: k8s.gcr.io/kube-cross:v1.12.12-1
Step #1: Setting up environment...
Step #1: Checking dependencies...
Step #1: Compiling github.com/psampaz/go-mod-outdated...
Step #1: go: finding github.com/psampaz/go-mod-outdated v0.5.0
Step #1: go: finding github.com/sirupsen/logrus v1.4.2
Step #1: go: finding github.com/nozzle/throttler v0.0.0-20180817012639-2ea982251481
Step #1: go: finding cloud.google.com/go v0.44.3
Step #1: go: finding github.com/google/go-github/v29 v29.0.2
Step #1: go: finding sigs.k8s.io/yaml v1.1.0
Step #1: go: finding github.com/google/uuid v1.1.1
Step #1: go: finding gopkg.in/src-d/go-git.v4 v4.13.1
Step #1: go: finding k8s.io/utils v0.0.0-20200117235808-5f6fbceb4c31
Step #1: go: finding github.com/shurcooL/sanitized_anchor_name v1.0.0
Step #1: go: finding golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
Step #1: go: finding github.com/pkg/errors v0.8.1
Step #1: go: finding github.com/GoogleCloudPlatform/testgrid v0.0.1-alpha.4
Step #1: go: finding github.com/davecgh/go-spew v1.1.1
Step #1: go: finding github.com/blang/semver v3.5.1+incompatible
Step #1: go: finding golang.org/x/tools v0.0.0-20190729092621-ff9f1409240a
Step #1: go: finding github.com/google/pprof v0.0.0-20190515194954-54271f7e092f
Step #1: go: finding github.com/golang/protobuf v1.3.2
Step #1: go: finding golang.org/x/net v0.0.0-20190311183353-d8887717615a
Step #1: go: finding google.golang.org/grpc v1.21.1
Step #1: go: finding github.com/xanzy/ssh-agent v0.2.1
Step #1: go: finding cloud.google.com/go v0.26.0
Step #1: go: finding github.com/golang/protobuf v1.2.0
Step #1: go: finding github.com/pmezard/go-difflib v1.0.0
Step #1: go: finding golang.org/x/text v0.3.2
Step #1: go: finding github.com/src-d/gcfg v1.4.0
Step #1: go: finding golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a
Step #1: go: finding google.golang.org/genproto v0.0.0-20190801165951-fa694d86fc64
Step #1: go: finding github.com/pelletier/go-buffruneio v0.2.0
Step #1: go: finding github.com/modern-go/reflect2 v1.0.1
Step #1: go: finding github.com/google/go-cmp v0.2.0
Step #1: go: finding golang.org/x/text v0.3.0
Step #1: go: finding gopkg.in/warnings.v0 v0.1.2
Step #1: go: finding golang.org/x/lint v0.0.0-20190409202823-959b441ac422
Step #1: go: finding golang.org/x/crypto v0.0.0-20190219172222-a4c6cb3142f2
Step #1: go: finding golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
Step #1: go: finding github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
Step #1: go: finding github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7
Step #1: go: finding google.golang.org/grpc v1.23.1
Step #1: go: finding github.com/google/go-cmp v0.3.1
Step #1: go: finding golang.org/x/tools v0.0.0-20190628153133-6cdbf07be9d0
Step #1: go: finding github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239
Step #1: go: finding github.com/stretchr/testify v1.2.2
Step #1: go: finding cloud.google.com/go v0.34.0
Step #1: go: finding google.golang.org/appengine v1.4.0
Step #1: go: finding golang.org/x/tools v0.0.0-20190226205152-f727befe758c
Step #1: go: finding go.opencensus.io v0.22.1
Step #1: go: finding github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
Step #1: go: finding honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a
Step #1: go: finding github.com/emirpasic/gods v1.12.0
Step #1: go: finding go.opencensus.io v0.22.0
Step #1: go: finding github.com/olekukonko/tablewriter v0.0.1
Step #1: go: finding golang.org/x/lint v0.0.0-20190227174305-5b3e6a55c961
Step #1: go: finding google.golang.org/api v0.8.0
Step #1: go: finding github.com/client9/misspell v0.3.4
Step #1: go: finding golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
Step #1: go: finding github.com/google/martian v2.1.0+incompatible
Step #1: go: finding rsc.io/binaryregexp v0.2.0
Step #1: go: finding github.com/spf13/cobra v0.0.5
Step #1: go: finding golang.org/x/sync v0.0.0-20190423024810-112230192c58
Step #1: go: finding golang.org/x/exp v0.0.0-20190510132918-efd6b22b2522
Step #1: go: finding github.com/stretchr/objx v0.2.0
Step #1: go: finding golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e
Step #1: go: finding cloud.google.com/go v0.38.0
Step #1: go: finding github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
Step #1: go: finding github.com/spf13/pflag v1.0.5
Step #1: go: finding github.com/mattn/go-runewidth v0.0.4
Step #1: go: finding golang.org/x/exp v0.0.0-20190121172915-509febef88a4
Step #1: go: finding github.com/sendgrid/rest v2.4.1+incompatible
Step #1: go: finding golang.org/x/sys v0.0.0-20190312061237-fead79001313
Step #1: go: finding github.com/inconshreveable/mousetrap v1.0.0
Step #1: go: finding github.com/google/btree v1.0.0
Step #1: go: finding golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
Step #1: go: finding golang.org/x/sys v0.0.0-20190422165155-953cdadca894
Step #1: go: finding google.golang.org/genproto v0.0.0-20190502173448-54afdca5d873
Step #1: go: finding github.com/jessevdk/go-flags v1.4.0
Step #1: go: finding golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
Step #1: go: finding google.golang.org/appengine v1.6.2
Step #1: go: finding golang.org/x/tools v0.0.0-20190312170243-e65039ee4138
Step #1: go: finding github.com/kisielk/errcheck v1.2.0
Step #1: go: finding github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c
Step #1: go: finding github.com/stretchr/testify v1.3.0
Step #1: go: finding github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802
Step #1: go: finding go.opencensus.io v0.21.0
Step #1: go: finding google.golang.org/appengine v1.1.0
Step #1: go: finding golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
Step #1: go: finding github.com/BurntSushi/toml v0.3.1
Step #1: go: finding gopkg.in/yaml.v2 v2.2.2
Step #1: go: finding golang.org/x/sync v0.0.0-20181108010431-42b317875d0f
Step #1: go: finding google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
Step #1: go: finding github.com/spf13/pflag v1.0.3
Step #1: go: finding k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
Step #1: go: finding gopkg.in/russross/blackfriday.v2 v2.0.0
Step #1: go: finding golang.org/x/image v0.0.0-20190227222117-0694c2d4d067
Step #1: go: finding golang.org/x/tools v0.0.0-20190606124116-d0a3d012864b
Step #1: go: finding github.com/kisielk/gotool v1.0.0
Step #1: go: finding honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc
Step #1: go: finding github.com/stretchr/testify v1.4.0
Step #1: go: finding golang.org/x/net v0.0.0-20190603091049-60506f45cf65
Step #1: go: finding github.com/sergi/go-diff v1.0.0
Step #1: go: finding golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
Step #1: go: finding github.com/spf13/viper v1.3.2
Step #1: go: finding github.com/konsorten/go-windows-terminal-sequences v1.0.1
Step #1: go: finding github.com/google/go-querystring v1.0.0
Step #1: go: finding golang.org/x/net v0.0.0-20190108225652-1e06a53dbb7e
Step #1: go: finding github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024
Step #1: go: finding google.golang.org/grpc v1.20.1
Step #1: go: finding github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99
Step #1: go: finding golang.org/x/tools v0.0.0-20190506145303-2d16b83fe98c
Step #1: go: finding google.golang.org/genproto v0.0.0-20190425155659-357c62f0e4bb
Step #1: go: finding golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5
Step #1: go: finding github.com/coreos/etcd v3.3.10+incompatible
Step #1: go: finding google.golang.org/genproto v0.0.0-20190418145605-e7d98fc518a7
Step #1: go: finding golang.org/x/tools v0.0.0-20190311212946-11955173bddd
Step #1: go: finding github.com/kr/pretty v0.1.0
Step #1: go: finding github.com/spf13/afero v1.2.2
Step #1: go: finding github.com/golang/mock v1.3.1
Step #1: go: finding github.com/coreos/go-semver v0.2.0
Step #1: go: finding golang.org/x/sys v0.0.0-20190606165138-5da285871e9c
Step #1: go: finding golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4
Step #1: go: finding github.com/googleapis/gax-go/v2 v2.0.5
Step #1: go: finding golang.org/x/sys v0.0.0-20190412213103-97732733099d
Step #1: go: finding github.com/hashicorp/golang-lru v0.5.0
Step #1: go: finding golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3
Step #1: go: finding honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099
Step #1: go: finding gopkg.in/inf.v0 v0.9.1
Step #1: go: finding golang.org/x/crypto v0.0.0-20181203042331-505ab145d0a9
Step #1: go: finding gopkg.in/src-d/go-git-fixtures.v3 v3.5.0
Step #1: go: finding google.golang.org/api v0.9.0
Step #1: go: finding golang.org/x/net v0.0.0-20180724234803-3673e40ba225
Step #1: go: finding golang.org/x/net v0.0.0-20190501004415-9ce7a6920f09
Step #1: go: finding k8s.io/klog v1.0.0
Step #1: go: finding github.com/hashicorp/hcl v1.0.0
Step #1: go: finding cloud.google.com/go/datastore v1.0.0
Step #1: go: finding golang.org/x/net v0.0.0-20190213061140-3a22650c66bd
Step #1: go: finding github.com/coreos/go-etcd v2.0.0+incompatible
Step #1: go: finding github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6
Step #1: go: finding github.com/google/go-cmp v0.3.0
Step #1: go: finding google.golang.org/genproto v0.0.0-20190307195333-5fe7a883aa19
Step #1: go: finding github.com/stretchr/objx v0.1.0
Step #1: go: finding gopkg.in/src-d/go-billy.v4 v4.3.2
Step #1: go: finding gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
Step #1: go: finding google.golang.org/appengine v1.5.0
Step #1: go: finding golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
Step #1: go: finding github.com/stretchr/objx v0.1.1
Step #1: go: finding github.com/spf13/jwalterweatherman v1.0.0
Step #1: go: finding google.golang.org/api v0.7.0
Step #1: go: finding github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77
Step #1: go: finding golang.org/x/tools v0.0.0-20190425150028-36563e24a262
Step #1: go: finding golang.org/x/sys v0.0.0-20190507160741-ecd444e8653b
Step #1: go: finding golang.org/x/sys v0.0.0-20181205085412-a5c9d58dba9a
Step #1: go: finding github.com/spf13/afero v1.1.2
Step #1: go: finding github.com/kr/text v0.1.0
Step #1: go: finding github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
Step #1: go: finding golang.org/x/sys v0.0.0-20190726091711-fc99dfbffb4e
Step #1: go: finding github.com/mitchellh/go-homedir v1.1.0
Step #1: go: finding google.golang.org/grpc v1.19.0
Step #1: go: finding golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
Step #1: go: finding golang.org/x/sys v0.0.0-20190502145724-3ef323f4f1fd
Step #1: go: finding github.com/google/pprof v0.0.0-20181206194817-3ea8567a2e57
Step #1: go: finding github.com/pelletier/go-toml v1.2.0
Step #1: go: finding gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
Step #1: go: finding github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1
Step #1: go: finding golang.org/x/sys v0.0.0-20190624142023-c5567b49c5d0
Step #1: go: finding github.com/kr/pty v1.1.1
Step #1: go: finding golang.org/x/tools v0.0.0-20181030221726-6c7e314b6563
Step #1: go: finding github.com/fsnotify/fsnotify v1.4.7
Step #1: go: finding golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297
Step #1: go: finding github.com/golang/mock v1.1.1
Step #1: go: finding golang.org/x/tools v0.0.0-20190312151545-0bb0c0a6e846
Step #1: go: finding github.com/gliderlabs/ssh v0.2.2
Step #1: go: finding github.com/golang/mock v1.2.0
Step #1: go: finding vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
Step #1: go: finding golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f
Step #1: go: finding golang.org/x/mobile v0.0.0-20190312151609-d3739f865fa6
Step #1: go: finding github.com/go-logr/logr v0.1.0
Step #1: go: finding golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
Step #1: go: finding golang.org/x/sys v0.0.0-20190221075227-b4e8571b14e0
Step #1: go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
Step #1: go: finding github.com/hashicorp/golang-lru v0.5.1
Step #1: go: finding google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
Step #1: go: finding github.com/golangci/golangci-lint v1.23.3
Step #1: go: finding golang.org/x/sys v0.0.0-20180830151530-49385e6e1522
Step #1: go: finding k8s.io/klog v0.3.0
Step #1: go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
Step #1: go: finding google.golang.org/api v0.4.0
Step #1: go: finding honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a
Step #1: go: finding golang.org/x/tools v0.0.0-20190114222345-bf090417da8b
Step #1: go: finding github.com/cpuguy83/go-md2man v1.0.10
Step #1: go: finding github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568
Step #1: go: finding github.com/googleapis/gax-go/v2 v2.0.4
Step #1: go: finding cloud.google.com/go v0.44.1
Step #1: go: finding github.com/golangci/check v0.0.0-20180506172741-cfe4005ccda2
Step #1: go: finding github.com/golangci/lint-1 v0.0.0-20191013205115-297bf364a8e0
Step #1: go: finding github.com/kr/pty v1.1.8
Step #1: go: finding github.com/golangci/misspell v0.0.0-20180809174111-950f5d19e770
Step #1: go: finding golang.org/x/net v0.0.0-20180826012351-8a410e7b638d
Step #1: go: finding github.com/golangci/gofmt v0.0.0-20190930125516-244bba706f1a
Step #1: go: finding github.com/golangci/dupl v0.0.0-20180902072040-3e9179ac440a
Step #1: go: finding honnef.co/go/tools v0.0.1-2019.2.3
Step #1: go: finding github.com/spf13/cast v1.3.0
Step #1: go: finding golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f
Step #1: go: finding golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c
Step #1: go: finding github.com/sourcegraph/go-diff v0.5.1
Step #1: go: finding github.com/golangci/unconvert v0.0.0-20180507085042-28b1c447d1f4
Step #1: go: finding github.com/shirou/gopsutil v0.0.0-20190901111213-e4ec7b275ada
Step #1: go: finding golang.org/x/tools v0.0.0-20200102140908-9497f49d5709
Step #1: go: finding golang.org/x/net v0.0.0-20190620200207-3b0461eec859
Step #1: go: finding github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8
Step #1: go: finding github.com/securego/gosec v0.0.0-20200103095621-79fbf3af8d83
Step #1: go: finding github.com/go-lintpack/lintpack v0.5.2
Step #1: go: finding github.com/google/renameio v0.1.0
Step #1: go: finding github.com/jirfag/go-printf-func-name v0.0.0-20191110105641-45db9963cdd3
Step #1: go: finding github.com/ultraware/whitespace v0.0.4
Step #1: go: finding github.com/russross/blackfriday v1.5.2
Step #1: go: finding golang.org/x/lint v0.0.0-20181026193005-c67002cb31c3
Step #1: go: finding github.com/go-critic/go-critic v0.4.1
Step #1: go: finding github.com/matoous/godox v0.0.0-20190911065817-5d6d842e92eb
Step #1: go: finding github.com/rogpeppe/go-internal v1.3.0
Step #1: go: finding github.com/shirou/w32 v0.0.0-20160930032740-bb4de0191aa4
Step #1: go: finding github.com/shurcooL/go v0.0.0-20180423040247-9e1955d9fb6e
Step #1: go: finding github.com/golangci/goconst v0.0.0-20180610141641-041c5f2b40f3
Step #1: go: finding github.com/sendgrid/sendgrid-go v3.5.0+incompatible
Step #1: go: finding github.com/spf13/viper v1.6.1
Step #1: go: finding github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
Step #1: go: finding golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135
Step #1: go: finding mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed
Step #1: go: finding github.com/go-toolsmith/pkgload v0.0.0-20181119091011-e9e65178eee8
Step #1: go: finding github.com/nbutton23/zxcvbn-go v0.0.0-20180912185939-ae427f1e4c1d
Step #1: go: finding github.com/gofrs/flock v0.0.0-20190320160742-5135e617513b
Step #1: go: finding golang.org/x/tools v0.0.0-20190621195816-6e04913cbbac
Step #1: go: finding github.com/go-toolsmith/astcopy v1.0.0
Step #1: go: finding github.com/mitchellh/go-ps v0.0.0-20190716172923-621e5597135b
Step #1: go: finding golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
Step #1: go: finding github.com/golangci/errcheck v0.0.0-20181223084120-ef45e06d44b6
Step #1: go: finding github.com/pborman/uuid v1.2.0
Step #1: go: finding github.com/ultraware/funlen v0.0.2
Step #1: go: finding github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
Step #1: go: finding github.com/mozilla/tls-observatory v0.0.0-20190404164649-a3c1b6cfecfd
Step #1: go: finding go.uber.org/multierr v1.1.0
Step #1: go: finding github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
Step #1: go: finding github.com/go-toolsmith/astfmt v1.0.0
Step #1: go: finding github.com/golangci/maligned v0.0.0-20180506175553-b1d89398deca
Step #1: go: finding github.com/jonboulle/clockwork v0.1.0
Step #1: go: finding github.com/coreos/bbolt v1.3.2
Step #1: go: finding github.com/valyala/quicktemplate v1.2.0
Step #1: go: finding github.com/jingyugao/rowserrcheck v0.0.0-20191204022205-72ab7603b68a
Step #1: go: finding github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
Step #1: go: finding gopkg.in/yaml.v2 v2.2.4
Step #1: go: finding github.com/golang/protobuf v1.3.1
Step #1: go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
Step #1: go: finding go.uber.org/zap v1.10.0
Step #1: go: finding github.com/go-toolsmith/astp v1.0.0
Step #1: go: finding golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898
Step #1: go: finding github.com/mattn/go-colorable v0.1.4
Step #1: go: finding github.com/golangci/prealloc v0.0.0-20180630174525-215b22d4de21
Step #1: go: finding golang.org/x/tools v0.0.0-20190521203540-521d6ed310dd
Step #1: go: finding github.com/tommy-muehle/go-mnd v1.1.1
Step #1: go: finding github.com/klauspost/compress v1.4.1
Step #1: go: finding github.com/OpenPeeDeeP/depguard v1.0.1
Step #1: go: finding github.com/grpc-ecosystem/grpc-gateway v1.9.0
Step #1: go: finding google.golang.org/appengine v1.6.1
Step #1: go: finding github.com/valyala/bytebufferpool v1.0.0
Step #1: go: finding github.com/jmoiron/sqlx v1.2.1-0.20190826204134-d7d95172beb5
Step #1: go: finding github.com/shurcooL/go-goon v0.0.0-20170922171312-37c2f522c041
Step #1: go: finding gopkg.in/errgo.v2 v2.1.0
Step #1: go: finding github.com/ugorji/go v1.1.4
Step #1: go: finding golang.org/x/mod v0.1.1-0.20191105210325-c90efee705ee
Step #1: go: finding golang.org/x/tools v0.0.0-20191108193012-7d206e10da11
Step #1: go: finding golang.org/x/tools v0.0.0-20190910044552-dd2b5c81c578
Step #1: go: finding github.com/gobwas/glob v0.2.3
Step #1: go: finding github.com/golangci/go-misc v0.0.0-20180628070357-927a3d87b613
Step #1: go: finding gopkg.in/ini.v1 v1.51.0
Step #1: go: finding github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
Step #1: go: finding github.com/klauspost/cpuid v1.2.0
Step #1: go: finding github.com/gostaticanalysis/analysisutil v0.0.0-20190318220348-4088753ea4d3
Step #1: go: finding sourcegraph.com/sqs/pbtypes v0.0.0-20180604144634-d3ebe8f20ae4
Step #1: go: finding github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c
Step #1: go: finding gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
Step #1: go: finding go.uber.org/atomic v1.4.0
Step #1: go: finding github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6
Step #1: go: finding github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
Step #1: go: finding github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef
Step #1: go: finding golang.org/x/time v0.0.0-20181108054448-85acf8d2951c
Step #1: go: finding github.com/smartystreets/goconvey v1.6.4
Step #1: go: finding golang.org/x/net v0.0.0-20190923162816-aa69164e4478
Step #1: go: finding github.com/gogo/protobuf v1.2.1
Step #1: go: finding golang.org/x/net v0.0.0-20181220203305-927f97764cc3
Step #1: go: finding github.com/logrusorgru/aurora v0.0.0-20181002194514-a7b3b318ed4e
Step #1: go: finding github.com/valyala/fasthttp v1.2.0
Step #1: go: finding golang.org/x/sys v0.0.0-20180909124046-d0be0721c37e
Step #1: go: finding golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb
Step #1: go: finding github.com/lib/pq v1.2.0
Step #1: go: finding github.com/fatih/color v1.7.0
Step #1: go: finding golang.org/x/tools v0.0.0-20190221204921-83362c3779f5
Step #1: go: finding github.com/go-toolsmith/astinfo v0.0.0-20180906194353-9809ff7efb21
Step #1: go: finding github.com/onsi/ginkgo v1.8.0
Step #1: go: finding github.com/mattn/go-isatty v0.0.8
Step #1: go: finding github.com/google/uuid v1.0.0
Step #1: go: finding golang.org/x/tools v0.0.0-20190328211700-ab21143f2384
Step #1: go: finding github.com/davecgh/go-spew v1.1.0
Step #1: go: finding go.etcd.io/bbolt v1.3.2
Step #1: go: finding github.com/go-sql-driver/mysql v1.4.0
Step #1: go: finding github.com/go-toolsmith/astequal v1.0.0
Step #1: go: finding github.com/golangci/ineffassign v0.0.0-20190609212857-42439a7714cc
Step #1: go: finding github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2
Step #1: go: finding github.com/magiconair/properties v1.8.0
Step #1: go: finding github.com/go-toolsmith/astp v0.0.0-20180903215135-0af7e3c24f30
Step #1: go: finding github.com/mitchellh/mapstructure v1.1.2
Step #1: go: finding github.com/onsi/gomega v1.8.1
Step #1: go: finding golang.org/x/net v0.0.0-20180911220305-26e67e76b6c3
Step #1: go: finding github.com/gogo/protobuf v1.1.1
Step #1: go: finding github.com/klauspost/cpuid v0.0.0-20180405133222-e7e905edc00e
Step #1: go: finding github.com/golangci/revgrep v0.0.0-20180526074752-d9c87f5ffaf0
Step #1: go: finding github.com/subosito/gotenv v1.2.0
Step #1: go: finding golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
Step #1: go: finding gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
Step #1: go: finding github.com/gorilla/websocket v1.4.0
Step #1: go: finding github.com/sclevine/spec v1.2.0
Step #1: go: finding github.com/go-toolsmith/strparse v1.0.0
Step #1: go: finding golang.org/x/sys v0.0.0-20190222072716-a9d3bda3a223
Step #1: go: finding mvdan.cc/unparam v0.0.0-20190720180237-d51796306d8f
Step #1: go: finding gopkg.in/resty.v1 v1.12.0
Step #1: go: finding gopkg.in/yaml.v2 v2.2.7
Step #1: go: finding golang.org/x/tools v0.0.0-20190110163146-51295c7ec13a
Step #1: go: finding github.com/hpcloud/tail v1.0.0
Step #1: go: finding github.com/magiconair/properties v1.8.1
Step #1: go: finding github.com/onsi/ginkgo v1.11.0
Step #1: go: finding github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
Step #1: go: finding golang.org/x/net v0.0.0-20190628185345-da137c7871d7
Step #1: go: finding github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1
Step #1: go: finding google.golang.org/grpc v1.21.0
Step #1: go: finding github.com/go-toolsmith/astcast v1.0.0
Step #1: go: finding github.com/go-toolsmith/astfmt v0.0.0-20180903215011-8f8ee99c3086
Step #1: go: finding github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d
Step #1: go: finding github.com/go-toolsmith/astequal v0.0.0-20180903214952-dcb477bfacd6
Step #1: go: finding github.com/joefitzgerald/rainbow-reporter v0.1.0
Step #1: go: finding github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
Step #1: go: finding golang.org/x/tools v0.0.0-20190311215038-5c2858a9cfe5
Step #1: go: finding github.com/bombsimon/wsl/v2 v2.0.0
Step #1: go: finding golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
Step #1: go: finding golang.org/x/tools v0.0.0-20190719005602-e377ae9d6386
Step #1: go: finding gopkg.in/fsnotify.v1 v1.4.7
Step #1: go: finding github.com/lib/pq v1.0.0
Step #1: go: finding github.com/timakin/bodyclose v0.0.0-20190930140734-f7f2e9bca95e
Step #1: go: finding golang.org/x/sys v0.0.0-20181107165924-66b7b1311ac8
Step #1: go: finding golang.org/x/tools v0.0.0-20190322203728-c1a832b0ad89
Step #1: go: finding github.com/uudashr/gocognit v1.0.1
Step #1: go: finding mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b
Step #1: go: finding github.com/go-toolsmith/pkgload v1.0.0
Step #1: go: finding github.com/onsi/gomega v1.5.0
Step #1: go: finding github.com/go-ole/go-ole v1.2.1
Step #1: go: finding golang.org/x/tools v0.0.0-20180221164845-07fd8470d635
Step #1: go: finding golang.org/x/mod v0.0.0-20190513183733-4bf6d317e70e
Step #1: go: finding github.com/json-iterator/go v1.1.7
Step #1: go: finding github.com/kisielk/errcheck v1.1.0
Step #1: go: finding golang.org/x/tools v0.0.0-20181117154741-2ddaf7f79a09
Step #1: go: finding golang.org/x/tools v0.0.0-20191113232020-e2727e816f5a
Step #1: go: finding github.com/mattn/go-sqlite3 v1.9.0
Step #1: go: finding github.com/rogpeppe/fastuuid v0.0.0-20150106093220-6724a57986af
Step #1: go: finding github.com/go-toolsmith/typep v1.0.0
Step #1: go: finding github.com/prometheus/client_golang v0.9.3
Step #1: go: finding golang.org/x/tools v0.0.0-20180525024113-a5b4c53f6e8b
Step #1: go: finding github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742
Step #1: go: finding github.com/jtolds/gls v4.20.0+incompatible
Step #1: go: finding github.com/mattn/goveralls v0.0.2
Step #1: go: finding github.com/ghodss/yaml v1.0.0
Step #1: go: finding gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
Step #1: go: finding golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
Step #1: go: finding github.com/soheilhy/cmux v0.1.4
Step #1: go: finding golang.org/x/net v0.0.0-20190522155817-f3200d17e092
Step #1: go: finding gopkg.in/yaml.v2 v2.2.1
Step #1: go: finding github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084
Step #1: go: finding github.com/prometheus/common v0.4.0
Step #1: go: finding github.com/prometheus/tsdb v0.7.1
Step #1: go: finding github.com/creack/pty v1.1.7
Step #1: go: finding github.com/modern-go/concurrent v0.0.0-20180228061459-e0a39a4cb421
Step #1: go: finding golang.org/x/tools v0.0.0-20190706070813-72ffa07ba3db
Step #1: go: finding github.com/golangci/gocyclo v0.0.0-20180528134321-2becd97e67ee
Step #1: go: finding github.com/klauspost/compress v1.4.0
Step #1: go: finding github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
Step #1: go: finding github.com/go-logfmt/logfmt v0.4.0
Step #1: go: finding github.com/julienschmidt/httprouter v1.2.0
Step #1: go: finding gopkg.in/alecthomas/kingpin.v2 v2.2.6
Step #1: go: finding github.com/prometheus/client_model v0.0.0-20180712105110-5c3871d89910
Step #1: go: finding golang.org/x/net v0.0.0-20180906233101-161cd47e91fd
Step #1: go: finding github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d
Step #1: go: finding golang.org/x/sys v0.0.0-20181116152217-5ac8a444bdc5
Step #1: go: finding github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc
Step #1: go: finding github.com/prometheus/client_golang v0.9.1
Step #1: go: finding github.com/beorn7/perks v1.0.0
Step #1: go: finding golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529
Step #1: go: finding github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
Step #1: go: finding github.com/go-logfmt/logfmt v0.3.0
Step #1: go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
Step #1: go: finding gopkg.in/yaml.v2 v2.2.5
Step #1: go: finding github.com/google/gofuzz v1.0.0
Step #1: go: finding github.com/go-kit/kit v0.8.0
Step #1: go: finding github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223
Step #1: go: finding github.com/go-stack/stack v1.8.0
Step #1: go: finding golang.org/x/net v0.0.0-20181114220301-adae6a3d119a
Step #1: go: finding github.com/pkg/errors v0.8.0
Step #1: go: finding github.com/oklog/ulid v1.3.1
Step #1: go: finding github.com/cespare/xxhash v1.1.0
Step #1: go: finding github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954
Step #1: go: finding github.com/sirupsen/logrus v1.2.0
Step #1: go: finding github.com/prometheus/common v0.0.0-20181113130724-41aa239b4cce
Step #1: go: finding github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf
Step #1: go: finding github.com/onsi/ginkgo v1.6.0
Step #1: go: finding github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
Step #1: go: finding github.com/OneOfOne/xxhash v1.2.2
Step #1: go: finding github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72
Step #1: go: finding golang.org/x/crypto v0.0.0-20180904163835-0709b304e793
Step #1: go: finding golang.org/x/sys v0.0.0-20180905080454-ebe1bf3edb33
Step #1: go: downloading github.com/psampaz/go-mod-outdated v0.5.0
Step #1: go: extracting github.com/psampaz/go-mod-outdated v0.5.0
Step #1: go: downloading github.com/olekukonko/tablewriter v0.0.1
Step #1: go: extracting github.com/olekukonko/tablewriter v0.0.1
Step #1: go: downloading github.com/mattn/go-runewidth v0.0.4
Step #1: go: extracting github.com/mattn/go-runewidth v0.0.4
Step #1: github.com/psampaz/go-mod-outdated compiled & installed
Step #1: go: finding github.com/kevinburke/ssh_config latest
Step #1: go: finding github.com/golangci/goconst latest
Step #1: go: finding github.com/mattn/go-isatty v0.0.12
Step #1: go: finding github.com/golangci/gofmt latest
Step #1: go: finding github.com/spf13/viper v1.6.2
Step #1: go: finding github.com/prometheus/common v0.9.1
Step #1: go: finding github.com/golangci/prealloc latest
Step #1: go: finding github.com/xordataexchange/crypt v0.0.2
Step #1: go: finding github.com/BurntSushi/xgb latest
Step #1: go: finding golang.org/x/crypto latest
Step #1: go: finding golang.org/x/mod v0.2.0
Step #1: go: finding github.com/coreos/pkg latest
Step #1: go: finding gopkg.in/tomb.v1 latest
Step #1: go: finding go.opencensus.io v0.22.3
Step #1: go: finding github.com/creack/pty v1.1.9
Step #1: go: finding github.com/mattn/go-runewidth v0.0.8
Step #1: go: finding github.com/alcortesm/tgz latest
Step #1: go: finding golang.org/x/oauth2 latest
Step #1: go: finding mvdan.cc/interfacer latest
Step #1: go: finding github.com/golangci/golangci-lint v1.23.6
Step #1: go: finding github.com/shurcooL/go-goon latest
Step #1: go: finding github.com/olekukonko/tablewriter v0.0.4
Step #1: go: finding github.com/jirfag/go-printf-func-name latest
Step #1: go: finding github.com/flynn/go-shlex latest
Step #1: go: finding gopkg.in/ini.v1 v1.52.0
Step #1: go: finding github.com/xiang90/probing latest
Step #1: go: finding github.com/prometheus/client_model v0.2.0
Step #1: go: finding github.com/coreos/etcd v3.3.18+incompatible
Step #1: go: finding github.com/golang/groupcache latest
Step #1: go: finding github.com/spf13/jwalterweatherman v1.1.0
Step #1: go: finding github.com/golangci/lint-1 latest
Step #1: go: finding github.com/golangci/unconvert latest
Step #1: go: finding github.com/mwitkow/go-conntrack latest
Step #1: go: finding github.com/tommy-muehle/go-mnd v1.2.0
Step #1: go: finding github.com/onsi/gomega v1.9.0
Step #1: go: finding github.com/google/gofuzz v1.1.0
Step #1: go: finding github.com/mozilla/tls-observatory latest
Step #1: go: finding github.com/GoogleCloudPlatform/testgrid v0.0.1
Step #1: go: finding github.com/dgryski/go-sip13 latest
Step #1: go: finding github.com/gogo/protobuf v1.3.1
Step #1: go: finding github.com/alecthomas/template latest
Step #1: go: finding github.com/shurcooL/go latest
Step #1: go: finding github.com/jmoiron/sqlx v1.2.0
Step #1: go: finding github.com/securego/gosec latest
Step #1: go: finding gopkg.in/check.v1 latest
Step #1: go: finding k8s.io/apimachinery v0.17.2
Step #1: go: finding github.com/google/go-cmp v0.4.0
Step #1: go: finding github.com/golang/protobuf v1.3.3
Step #1: go: finding github.com/OneOfOne/xxhash v1.2.7
Step #1: go: finding gopkg.in/yaml.v2 v2.2.8
Step #1: go: finding golang.org/x/xerrors latest
Step #1: go: finding github.com/prometheus/procfs v0.0.8
Step #1: go: finding github.com/rogpeppe/go-internal v1.5.2
Step #1: go: finding github.com/coreos/bbolt v1.3.3
Step #1: go: finding github.com/gostaticanalysis/analysisutil v0.0.3
Step #1: go: finding github.com/jstemmer/go-junit-report v0.9.1
Step #1: go: finding github.com/klauspost/cpuid v1.2.3
Step #1: go: finding golang.org/x/sync latest
Step #1: go: finding github.com/valyala/tcplisten latest
Step #1: go: finding github.com/mitchellh/go-ps v1.0.0
Step #1: go: finding golang.org/x/net latest
Step #1: go: finding github.com/rogpeppe/fastuuid v1.2.0
Step #1: go: finding mvdan.cc/unparam latest
Step #1: go: finding github.com/golangci/misspell v0.3.4
Step #1: go: finding github.com/hashicorp/golang-lru v0.5.4
Step #1: go: finding github.com/lib/pq v1.3.0
Step #1: go: finding github.com/mattn/go-sqlite3 v2.0.3+incompatible
Step #1: go: finding github.com/julienschmidt/httprouter v1.3.0
Step #1: go: finding github.com/jbenet/go-context latest
Step #1: go: finding github.com/go-toolsmith/astinfo v1.0.0
Step #1: go: finding go.uber.org/multierr v1.4.0
Step #1: go: finding github.com/konsorten/go-windows-terminal-sequences v1.0.2
Step #1: go: finding google.golang.org/grpc v1.27.1
Step #1: go: finding golang.org/x/mobile latest
Step #1: go: finding google.golang.org/genproto latest
Step #1: go: finding github.com/jingyugao/rowserrcheck latest
Step #1: go: finding github.com/xlab/handysort latest
Step #1: go: finding github.com/shirou/w32 latest
Step #1: go: finding github.com/ugorji/go v1.1.7
Step #1: go: finding github.com/golangci/go-misc latest
Step #1: go: finding github.com/go-kit/kit v0.9.0
Step #1: go: finding github.com/golangci/revgrep latest
Step #1: go: finding golang.org/x/image latest
Step #1: go: finding github.com/mattn/goveralls v0.0.5
Step #1: go: finding github.com/pelletier/go-toml v1.6.0
Step #1: go: finding golang.org/x/lint latest
Step #1: go: finding github.com/golangci/check latest
Step #1: go: finding github.com/anmitsu/go-shlex latest
Step #1: go: finding github.com/logrusorgru/aurora latest
Step #1: go: finding github.com/golangci/gocyclo latest
Step #1: go: finding github.com/google/pprof latest
Step #1: go: finding github.com/modern-go/concurrent latest
Step #1: go: finding github.com/google/go-github/v29 v29.0.3
Step #1: go: finding sourcegraph.com/sqs/pbtypes v1.0.0
Step #1: go: finding github.com/spaolacci/murmur3 v1.1.0
Step #1: go: finding github.com/smartystreets/assertions v1.0.1
Step #1: go: finding golang.org/x/tools latest
Step #1: go: finding github.com/gofrs/flock v0.7.1
Step #1: go: finding cloud.google.com/go v0.52.0
Step #1: go: finding github.com/sergi/go-diff v1.1.0
Step #1: go: finding github.com/go-ole/go-ole v1.2.4
Step #1: go: finding github.com/go-sql-driver/mysql v1.5.0
Step #1: go: finding k8s.io/utils latest
Step #1: go: finding github.com/armon/consul-api latest
Step #1: go: finding google.golang.org/api v0.17.0
Step #1: go: finding github.com/matoous/godox latest
Step #1: go: finding github.com/golangci/dupl latest
Step #1: go: finding github.com/pkg/errors v0.9.1
Step #1: go: finding google.golang.org/appengine v1.6.5
Step #1: go: finding github.com/json-iterator/go v1.1.9
Step #1: go: finding sigs.k8s.io/yaml v1.2.0
Step #1: go: finding github.com/golang/mock v1.4.0
Step #1: go: finding github.com/golangci/errcheck latest
Step #1: go: finding github.com/armon/go-socks5 latest
Step #1: go: finding github.com/go-logfmt/logfmt v0.5.0
Step #1: go: finding github.com/kr/logfmt latest
Step #1: go: finding github.com/spf13/cast v1.3.1
Step #1: go: finding github.com/alecthomas/units latest
Step #1: go: finding github.com/valyala/quicktemplate v1.4.1
Step #1: go: finding github.com/tmc/grpc-websocket-proxy latest
Step #1: go: finding github.com/nozzle/throttler latest
Step #1: go: finding github.com/gopherjs/gopherjs latest
Step #1: go: finding mvdan.cc/lint latest
Step #1: go: finding github.com/klauspost/compress v1.10.0
Step #1: go: finding github.com/onsi/ginkgo v1.12.0
Step #1: go: finding github.com/shirou/gopsutil v2.20.1+incompatible
Step #1: go: finding github.com/gorilla/websocket v1.4.1
Step #1: go: finding github.com/timakin/bodyclose latest
Step #1: go: finding github.com/golang/glog latest
Step #1: go: finding github.com/valyala/fasthttp v1.9.0
Step #1: go: finding github.com/StackExchange/wmi latest
Step #1: go: finding github.com/beorn7/perks v1.0.1
Step #1: go: finding go.uber.org/atomic v1.5.1
Step #1: go: finding github.com/nbutton23/zxcvbn-go latest
Step #1: go: finding github.com/prometheus/tsdb v0.10.0
Step #1: go: finding github.com/grpc-ecosystem/grpc-gateway v1.13.0
Step #1: go: finding github.com/golangci/ineffassign latest
Step #1: go: finding github.com/russross/blackfriday v2.0.0+incompatible
Step #1: go: finding github.com/sclevine/spec v1.4.0
Step #1: go: finding go.etcd.io/bbolt v1.3.3
Step #1: go: finding golang.org/x/exp latest
Step #1: go: finding github.com/golangci/maligned latest
Step #1: go: finding github.com/coreos/go-semver v0.3.0
Step #1: go: finding github.com/prometheus/client_golang v1.4.1
Step #1: go: finding github.com/quasilyte/go-consistent latest
Step #1: go: finding golang.org/x/time latest
Step #1: go: finding github.com/fatih/color v1.9.0
Step #1: go: finding github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
Step #1: go: finding github.com/kr/pretty v0.2.0
Step #1: go: finding golang.org/x/sys latest
Step #1: go: finding go.uber.org/zap v1.13.0
Step #1: go: finding vbom.ml/util latest
Step #1: +-----------------------------------------+------------------------------------+------------------------------------+--------+------------------+
Step #1: |                 MODULE                  |              VERSION               |            NEW VERSION             | DIRECT | VALID TIMESTAMPS |
Step #1: +-----------------------------------------+------------------------------------+------------------------------------+--------+------------------+
Step #1: | cloud.google.com/go                     | v0.44.3                            | v0.52.0                            | true   | true             |
Step #1: | github.com/GoogleCloudPlatform/testgrid | v0.0.1-alpha.4                     | v0.0.1                             | true   | true             |
Step #1: | github.com/golangci/golangci-lint       | v1.23.3                            | v1.23.6                            | true   | true             |
Step #1: | github.com/google/go-github/v29         | v29.0.2                            | v29.0.3                            | true   | true             |
Step #1: | github.com/pkg/errors                   | v0.8.1                             | v0.9.1                             | true   | true             |
Step #1: | k8s.io/utils                            | v0.0.0-20200117235808-5f6fbceb4c31 | v0.0.0-20200124190032-861946025e34 | true   | true             |
Step #1: | sigs.k8s.io/yaml                        | v1.1.0                             | v1.2.0                             | true   | true             |
Step #1: +-----------------------------------------+------------------------------------+------------------------------------+--------+------------------+
Step #1: Compiling default release tools...
Step #1: go: downloading github.com/GoogleCloudPlatform/testgrid v0.0.1-alpha.4
Step #1: Fetching https://proxy.golang.org/github.com/%21google%21cloud%21platform/testgrid/@v/v0.0.1-alpha.4.zip
Step #1: go: extracting github.com/GoogleCloudPlatform/testgrid v0.0.1-alpha.4
Step #1: go: downloading github.com/golang/protobuf v1.3.2
Step #1: go: downloading cloud.google.com/go v0.44.3
Step #1: Fetching https://proxy.golang.org/github.com/golang/protobuf/@v/v1.3.2.zip
Step #1: Fetching https://proxy.golang.org/cloud.google.com/go/@v/v0.44.3.zip
Step #1: go: downloading google.golang.org/api v0.9.0
Step #1: go: downloading vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
Step #1: Fetching https://proxy.golang.org/vbom.ml/util/@v/v0.0.0-20180919145318-efcd4e0f9787.zip
Step #1: Fetching https://proxy.golang.org/google.golang.org/api/@v/v0.9.0.zip
Step #1: go: extracting vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
Step #1: go: extracting github.com/golang/protobuf v1.3.2
Step #1: go: extracting cloud.google.com/go v0.44.3
Step #1: go: downloading github.com/googleapis/gax-go/v2 v2.0.5
Step #1: go: downloading google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
Step #1: Fetching https://proxy.golang.org/google.golang.org/genproto/@v/v0.0.0-20190819201941-24fa4b261c55.zip
Step #1: Fetching https://proxy.golang.org/github.com/googleapis/gax-go/v2/@v/v2.0.5.zip
Step #1: go: downloading google.golang.org/grpc v1.23.1
Step #1: Fetching https://proxy.golang.org/google.golang.org/grpc/@v/v1.23.1.zip
Step #1: go: extracting github.com/googleapis/gax-go/v2 v2.0.5
Step #1: go: extracting google.golang.org/grpc v1.23.1
Step #1: go: downloading go.opencensus.io v0.22.1
Step #1: Fetching https://proxy.golang.org/go.opencensus.io/@v/v0.22.1.zip
Step #1: go: downloading golang.org/x/net v0.0.0-20190923162816-aa69164e4478
Step #1: Fetching https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190923162816-aa69164e4478.zip
Step #1: go: extracting go.opencensus.io v0.22.1
Step #1: go: downloading github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
Step #1: Fetching https://proxy.golang.org/github.com/golang/groupcache/@v/v0.0.0-20190702054246-869f871628b6.zip
Step #1: go: extracting github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
Step #1: go: extracting golang.org/x/net v0.0.0-20190923162816-aa69164e4478
Step #1: go: downloading golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
Step #1: Fetching https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20190922100055-0a153f010e69.zip
Step #1: go: downloading golang.org/x/text v0.3.2
Step #1: Fetching https://proxy.golang.org/golang.org/x/text/@v/v0.3.2.zip
Step #1: go: extracting google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
Step #1: go: extracting golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
Step #1: go: extracting golang.org/x/text v0.3.2
Step #1: go: extracting google.golang.org/api v0.9.0
Step #1: go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
Step #1: Fetching https://proxy.golang.org/golang.org/x/oauth2/@v/v0.0.0-20200107190931-bf48bf16ab8d.zip
Step #1: go: extracting golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
Step #1: github.com/golang/groupcache/lru
Step #1: go.opencensus.io/trace/internal
Step #1: vbom.ml/util/sortorder
Step #1: go.opencensus.io
Step #1: go.opencensus.io/internal/tagencoding
Step #1: google.golang.org/grpc/internal/grpcsync
Step #1: google.golang.org/grpc/encoding
Step #1: cloud.google.com/go/internal/version
Step #1: golang.org/x/text/transform
Step #1: github.com/GoogleCloudPlatform/testgrid/metadata
Step #1: google.golang.org/api/googleapi/internal/uritemplates
Step #1: google.golang.org/grpc/internal/grpcrand
Step #1: google.golang.org/grpc/keepalive
Step #1: google.golang.org/grpc/internal/envconfig
Step #1: golang.org/x/sys/unix
Step #1: golang.org/x/net/internal/timeseries
Step #1: google.golang.org/grpc/metadata
Step #1: google.golang.org/grpc/grpclog
Step #1: google.golang.org/grpc/codes
Step #1: google.golang.org/grpc/tap
Step #1: golang.org/x/net/http2/hpack
Step #1: cloud.google.com/go/internal/optional
Step #1: go.opencensus.io/internal
Step #1: golang.org/x/text/unicode/bidi
Step #1: go.opencensus.io/trace/tracestate
Step #1: google.golang.org/api/iterator
Step #1: go.opencensus.io/resource
Step #1: github.com/golang/protobuf/proto
Step #1: google.golang.org/grpc/credentials/internal
Step #1: golang.org/x/oauth2/jws
Step #1: go.opencensus.io/tag
Step #1: github.com/GoogleCloudPlatform/testgrid/metadata/junit
Step #1: google.golang.org/grpc/internal/backoff
Step #1: golang.org/x/net/context/ctxhttp
Step #1: google.golang.org/api/googleapi/transport
Step #1: cloud.google.com/go/compute/metadata
Step #1: google.golang.org/api/googleapi
Step #1: google.golang.org/grpc/internal/balancerload
Step #1: google.golang.org/grpc/stats
Step #1: go.opencensus.io/trace
Step #1: golang.org/x/text/unicode/norm
Step #1: go.opencensus.io/metric/metricdata
Step #1: google.golang.org/grpc/connectivity
Step #1: google.golang.org/grpc/naming
Step #1: golang.org/x/net/trace
Step #1: golang.org/x/oauth2/internal
Step #1: google.golang.org/grpc/internal
Step #1: go.opencensus.io/stats/internal
Step #1: go.opencensus.io/metric/metricproducer
Step #1: google.golang.org/grpc/serviceconfig
Step #1: google.golang.org/grpc/resolver
Step #1: go.opencensus.io/stats
Step #1: golang.org/x/oauth2
Step #1: golang.org/x/text/secure/bidirule
Step #1: google.golang.org/grpc/resolver/dns
Step #1: google.golang.org/grpc/resolver/passthrough
Step #1: go.opencensus.io/stats/view
Step #1: go.opencensus.io/trace/propagation
Step #1: go.opencensus.io/plugin/ochttp/propagation/b3
Step #1: google.golang.org/api/transport/http/internal/propagation
Step #1: golang.org/x/oauth2/jwt
Step #1: golang.org/x/oauth2/google
Step #1: go.opencensus.io/plugin/ochttp
Step #1: golang.org/x/net/idna
Step #1: google.golang.org/grpc/internal/syscall
Step #1: golang.org/x/net/http/httpguts
Step #1: golang.org/x/net/http2
Step #1: google.golang.org/grpc/encoding/proto
Step #1: google.golang.org/grpc/credentials
Step #1: github.com/golang/protobuf/ptypes/any
Step #1: github.com/golang/protobuf/ptypes/duration
Step #1: github.com/golang/protobuf/ptypes/timestamp
Step #1: google.golang.org/genproto/googleapis/type/expr
Step #1: google.golang.org/genproto/googleapis/rpc/code
Step #1: github.com/GoogleCloudPlatform/testgrid/pb/test_status
Step #1: github.com/golang/protobuf/protoc-gen-go/descriptor
Step #1: github.com/GoogleCloudPlatform/testgrid/pb/custom_evaluator
Step #1: google.golang.org/genproto/googleapis/rpc/status
Step #1: google.golang.org/grpc/binarylog/grpc_binarylog_v1
Step #1: github.com/golang/protobuf/ptypes
Step #1: google.golang.org/grpc/balancer
Step #1: google.golang.org/grpc/peer
Step #1: google.golang.org/grpc/internal/channelz
Step #1: github.com/GoogleCloudPlatform/testgrid/pb/config
Step #1: google.golang.org/grpc/status
Step #1: google.golang.org/grpc/balancer/base
Step #1: cloud.google.com/go/internal/trace
Step #1: google.golang.org/grpc/internal/binarylog
Step #1: google.golang.org/grpc/balancer/roundrobin
Step #1: google.golang.org/genproto/googleapis/api/annotations
Step #1: google.golang.org/grpc/internal/transport
Step #1: google.golang.org/grpc
Step #1: github.com/googleapis/gax-go/v2
Step #1: google.golang.org/api/internal
Step #1: google.golang.org/genproto/googleapis/iam/v1
Step #1: cloud.google.com/go/internal
Step #1: google.golang.org/api/gensupport
Step #1: google.golang.org/api/option
Step #1: cloud.google.com/go/iam
Step #1: google.golang.org/api/transport/http
Step #1: google.golang.org/api/storage/v1
Step #1: cloud.google.com/go/storage
Step #1: github.com/GoogleCloudPlatform/testgrid/util/gcs
Step #1: github.com/GoogleCloudPlatform/testgrid/config
Step #1: k8s.io/release/cmd/blocking-testgrid-tests
Step #1: blocking-testgrid-tests was successfully compiled and installed to /workspace/bin/blocking-testgrid-tests
Step #1: Compiling default release tools...
Step #1: go: downloading github.com/sirupsen/logrus v1.4.2
Step #1: go: downloading github.com/pkg/errors v0.8.1
Step #1: go: downloading github.com/spf13/cobra v0.0.5
Step #1: Fetching https://proxy.golang.org/github.com/sirupsen/logrus/@v/v1.4.2.zip
Step #1: Fetching https://proxy.golang.org/github.com/pkg/errors/@v/v0.8.1.zip
Step #1: Fetching https://proxy.golang.org/github.com/spf13/cobra/@v/v0.0.5.zip
Step #1: go: downloading github.com/google/go-github/v29 v29.0.2
Step #1: go: downloading github.com/blang/semver v3.5.1+incompatible
Step #1: Fetching https://proxy.golang.org/github.com/google/go-github/v29/@v/v29.0.2.zip
Step #1: Fetching https://proxy.golang.org/github.com/blang/semver/@v/v3.5.1+incompatible.zip
Step #1: go: extracting github.com/pkg/errors v0.8.1
Step #1: go: extracting github.com/blang/semver v3.5.1+incompatible
Step #1: go: extracting github.com/sirupsen/logrus v1.4.2
Step #1: go: extracting github.com/spf13/cobra v0.0.5
Step #1: go: downloading github.com/spf13/pflag v1.0.5
Step #1: Fetching https://proxy.golang.org/github.com/spf13/pflag/@v/v1.0.5.zip
Step #1: go: extracting github.com/google/go-github/v29 v29.0.2
Step #1: go: extracting github.com/spf13/pflag v1.0.5
Step #1: go: downloading github.com/google/go-querystring v1.0.0
Step #1: go: downloading golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
Step #1: Fetching https://proxy.golang.org/github.com/google/go-querystring/@v/v1.0.0.zip
Step #1: Fetching https://proxy.golang.org/golang.org/x/crypto/@v/v0.0.0-20191011191535-87dc89f01550.zip
Step #1: go: extracting github.com/google/go-querystring v1.0.0
Step #1: go: extracting golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
Step #1: golang.org/x/crypto/cast5
Step #1: golang.org/x/crypto/openpgp/errors
Step #1: github.com/pkg/errors
Step #1: github.com/google/go-querystring/query
Step #1: github.com/blang/semver
Step #1: github.com/sirupsen/logrus
Step #1: golang.org/x/crypto/openpgp/elgamal
Step #1: golang.org/x/crypto/openpgp/s2k
Step #1: golang.org/x/crypto/openpgp/armor
Step #1: github.com/spf13/pflag
Step #1: golang.org/x/crypto/openpgp/packet
Step #1: k8s.io/release/pkg/command
Step #1: k8s.io/release/pkg/util
Step #1: # k8s.io/release/pkg/util
Step #1: pkg/util/common.go:114:34: syntax error: unexpected o755, expecting comma or )
Step #1: note: module requires Go 1.13
Step #1: # k8s.io/release/pkg/command
Step #1: pkg/command/command.go:110:22: x.String undefined (type *command has no field or method String)
Step #1: note: module requires Go 1.13
Step #1: golang.org/x/crypto/openpgp
Step #1: github.com/google/go-github/v29/github
Step #1: github.com/spf13/cobra
Finished Step #1
ERROR
ERROR: build step 1 "k8s.gcr.io/kube-cross:v1.12.12-1" failed: exit status 1
```
</details>

/assign @hasheddan @saschagrunert @cpanato 

**What you expected to happen**:
1.15 mock stage should have still been running/eventually succeed

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:

- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Others:

Looks like `pkg/command/command.go` needs fix as well:
```
Step #1: # k8s.io/release/pkg/command
Step #1: pkg/command/command.go:110:22: x.String undefined (type *command has no field or method String)
Step #1: note: module requires Go 1.13
```

--- a/cmd/krel/cmd/changelog.go
+++ b/cmd/krel/cmd/changelog.go
@@ -243,7 +243,7 @@ func writeMarkdown(repo *git.Repo, toc, markdown string, tag semver.Version) err
 		return ioutil.WriteFile(
 			changelogPath, []byte(strings.Join(
 				[]string{addTocMarkers(t), strings.TrimSpace(m)}, "\n",
-			)), 0o644,
+			)), os.FileMode(0644),
 		)
 	}
 
@@ -342,7 +342,7 @@ func writeHTML(tag semver.Version, markdown string) error {
 		return err
 	}
 	logrus.Infof("Writing single HTML to %s", absOutputPath)
-	return ioutil.WriteFile(absOutputPath, output.Bytes(), 0o644)
+	return ioutil.WriteFile(absOutputPath, output.Bytes(), os.FileMode(0644))
 }
 
 func lookupRemoteReleaseNotes(branch string) (string, error) {
@@ -444,7 +444,7 @@ func adaptChangelogReadmeFile(repo *git.Repo, tag semver.Version) error {
 
 	const nl = "\n"
 	if err := ioutil.WriteFile(
-		targetFile, []byte(strings.Join(res, nl)+nl), 0o644); err != nil {
+		targetFile, []byte(strings.Join(res, nl)+nl), os.FileMode(0644)); err != nil {
 		return errors.Wrap(err, "unable to write changelog README.md")
 	}
 	return nil

--- a/cmd/release-notes/main.go
+++ b/cmd/release-notes/main.go
@@ -240,7 +240,7 @@ func WriteReleaseNotes(releaseNotes notes.ReleaseNotes, history notes.ReleaseNot
 	var existingNotes notes.ReleaseNotes
 
 	if opts.Output != "" {
-		output, err = os.OpenFile(opts.Output, os.O_RDWR|os.O_CREATE, 0o644)
+		output, err = os.OpenFile(opts.Output, os.O_RDWR|os.O_CREATE, os.FileMode(0644))
 		if err != nil {
 			return errors.Wrapf(err, "opening the supplied output file")
 		}

--- a/pkg/command/command.go
+++ b/pkg/command/command.go
@@ -107,7 +107,15 @@ func (c *Command) RunSuccess() (err error) {
 func (c *Command) String() string {
 	str := []string{}
 	for _, x := range c.cmds {
-		str = append(str, x.String())
+		// Note: the following logic can be replaced with x.String(), which was
+		// implemented in go1.13
+		b := new(strings.Builder)
+		b.WriteString(x.Path)
+		for _, a := range x.Args[1:] {
+			b.WriteByte(' ')
+			b.WriteString(a)
+		}
+		str = append(str, b.String())
 	}
 	return strings.Join(str, " | ")
 }

--- a/pkg/git/git_integration_test.go
+++ b/pkg/git/git_integration_test.go
@@ -86,7 +86,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, testFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	worktree, err := cloneRepo.Worktree()
@@ -123,7 +123,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, branchTestFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 	_, err = worktree.Add(branchTestFileName)
 	require.Nil(t, err)
@@ -147,7 +147,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, secondBranchTestFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 	_, err = worktree.Add(secondBranchTestFileName)
 	require.Nil(t, err)
@@ -507,7 +507,7 @@ func TestCheckoutSuccess(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		testRepo.testFileName,
 		[]byte("hello world"),
-		0o644,
+		os.FileMode(0644),
 	))
 	res, err := command.NewWithWorkDir(
 		testRepo.sut.Dir(), "git", "diff", "--name-only").Run()
@@ -602,7 +602,7 @@ func TestRmSuccessForce(t *testing.T) {
 	testRepo := newTestRepo(t)
 	defer testRepo.cleanup(t)
 	require.Nil(t, ioutil.WriteFile(testRepo.testFileName,
-		[]byte("test"), 0o755),
+		[]byte("test"), os.FileMode(0755)),
 	)
 
 	require.Nil(t, testRepo.sut.Rm(true, testRepo.testFileName))
@@ -631,7 +631,7 @@ func TestRmFailureModified(t *testing.T) {
 	testRepo := newTestRepo(t)
 	defer testRepo.cleanup(t)
 	require.Nil(t, ioutil.WriteFile(testRepo.testFileName,
-		[]byte("test"), 0o755),
+		[]byte("test"), os.FileMode(0755)),
 	)
 	require.NotNil(t, testRepo.sut.Rm(false, testRepo.testFileName))
 }

--- a/pkg/kubepkg/kubepkg.go
+++ b/pkg/kubepkg/kubepkg.go
@@ -346,7 +346,7 @@ func (bc *buildConfig) run() error {
 		dstParts := []string{"bin", string(bc.Channel), fileName}
 
 		dstPath := filepath.Join(dstParts...)
-		if mkdirErr := os.MkdirAll(dstPath, 0o777); mkdirErr != nil {
+		if mkdirErr := os.MkdirAll(dstPath, os.FileMode(0777)); mkdirErr != nil {
 			return err
 		}
 

--- a/pkg/notes/client/record.go
+++ b/pkg/notes/client/record.go
@@ -21,6 +21,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"os"
 	"path/filepath"
 	"sync"
 
@@ -149,7 +150,7 @@ func (c *githubNotesRecordClient) recordAPICall(
 		return err
 	}
 	if err := ioutil.WriteFile(
-		filepath.Join(c.recordDir, fileName), file, 0o644,
+		filepath.Join(c.recordDir, fileName), file, os.FileMode(0644),
 	); err != nil {
 		return err
 	}

--- a/pkg/notes/document_test.go
+++ b/pkg/notes/document_test.go
@@ -74,7 +74,7 @@ func TestCreateDownloadsTable(t *testing.T) {
 		"kubernetes.tar.gz",
 	} {
 		require.Nil(t, ioutil.WriteFile(
-			filepath.Join(dir, file), []byte{1, 2, 3}, 0o644,
+			filepath.Join(dir, file), []byte{1, 2, 3}, os.FileMode(0644),
 		))
 	}
 

--- a/pkg/notes/options/options.go
+++ b/pkg/notes/options/options.go
@@ -173,7 +173,7 @@ func (o *Options) ValidateAndFinish() (err error) {
 	// Create the record dir
 	if o.RecordDir != "" {
 		logrus.Info("using record mode")
-		if err := os.MkdirAll(o.RecordDir, 0o755); err != nil {
+		if err := os.MkdirAll(o.RecordDir, os.FileMode(0755)); err != nil {
 			return err
 		}
 	}

--- a/pkg/notes/options/options_test.go
+++ b/pkg/notes/options/options_test.go
@@ -107,7 +107,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, testFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	worktree, err := cloneRepo.Worktree()
@@ -144,7 +144,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, branchTestFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 	_, err = worktree.Add(branchTestFileName)
 	require.Nil(t, err)
@@ -168,7 +168,7 @@ func newTestRepo(t *testing.T) *testRepo {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(cloneTempDir, secondBranchTestFileName),
 		[]byte("test-content"),
-		0o644,
+		os.FileMode(0644),
 	))
 	_, err = worktree.Add(secondBranchTestFileName)
 	require.Nil(t, err)

--- a/pkg/release/release_test.go
+++ b/pkg/release/release_test.go
@@ -51,13 +51,13 @@ func TestBuiltWithBazel(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		baseBazelFile,
 		[]byte("test"),
-		0o644,
+		os.FileMode(0644),
 	))
 	bazelFile := filepath.Join(bazelTmpDir, "bazel-bin/build/release-tars/kubernetes.tar.gz")
 	require.Nil(t, ioutil.WriteFile(
 		bazelFile,
 		[]byte("test"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	time.Sleep(1 * time.Second)
@@ -66,13 +66,13 @@ func TestBuiltWithBazel(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		baseDockerFile,
 		[]byte("test"),
-		0o644,
+		os.FileMode(0644),
 	))
 	dockerFile := filepath.Join(dockerTmpDir, "_output/release-tars/1.1.1.tar.gz")
 	require.Nil(t, ioutil.WriteFile(
 		dockerFile,
 		[]byte("test"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	defer cleanupTmps(t, baseTmpDir, bazelTmpDir, dockerTmpDir)
@@ -143,7 +143,7 @@ func TestReadBazelVersion(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		bazelVersionFile,
 		[]byte(version),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	defer cleanupTmps(t, baseTmpDir)
@@ -206,7 +206,7 @@ func TestReadDockerVersion(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		filepath.Join(baseTmpDir, dockerBuildPath, "kubernetes.tar.gz"),
 		b.Bytes(),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	defer cleanupTmps(t, baseTmpDir)

--- a/pkg/util/common.go
+++ b/pkg/util/common.go
@@ -111,7 +111,7 @@ func FakeGOPATH(srcDir string) (string, error) {
 	logrus.Debugf("GOPATH: %s", os.Getenv("GOPATH"))
 
 	gitRoot := fmt.Sprintf("%s/src/k8s.io", baseDir)
-	if err := os.MkdirAll(gitRoot, 0o755); err != nil {
+	if err := os.MkdirAll(gitRoot, os.FileMode(0755)); err != nil {
 		return "", err
 	}
 	gitRoot = filepath.Join(gitRoot, "kubernetes")

--- a/pkg/util/common_test.go
+++ b/pkg/util/common_test.go
@@ -39,7 +39,7 @@ func TestMoreRecent(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		testFileOne,
 		[]byte("file-one-contents"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	time.Sleep(1 * time.Second)
@@ -48,7 +48,7 @@ func TestMoreRecent(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		testFileTwo,
 		[]byte("file-two-contents"),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	notFile := filepath.Join(baseTmpDir, "noexist.txt")
@@ -152,7 +152,7 @@ func TestReadFileFromGzippedTar(t *testing.T) {
 	require.Nil(t, ioutil.WriteFile(
 		testTarPath,
 		b.Bytes(),
-		0o644,
+		os.FileMode(0644),
 	))
 
 	defer cleanupTmp(t, baseTmpDir)

